features/container/sentinel/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -51,9 +51,12 @@ featuresBoot = \
     wrap, \
     bundle/${karafVersion}, \
     config/${karafVersion}, \
-    kar/${karafVersion}, \
+    kar/${karafVersion} \
+    ), \
     scv-jceks-impl/${project.version}, \
     scv-shell/${project.version}, \
+    sentinel-core/${project.version},, \
```

Comment:
Since we are adding these two features by default on Sentinel, we should try to run full smoke suite before merging.  You need to push another branch with `smoke` in the branch name like `AR/jira/NMS-13837-smoke`. That will trigger full smoke suite run.  Once that passes we can merge this.

Reply:
Agree